### PR TITLE
feat: wrap v3

### PR DESCRIPTION
**Pull request by simboonlong**
- easy wrap v3 docs

### CODE REVIEW

File: src/tailwind.css [hunk 1]
```diff
@@ -13,9 +13,16 @@
...
-    @apply text-blue-700 border-blue-500 bg-transparent;
+  .btn-blue-small.selected {
+    @apply text-white bg-blue-500 hover:bg-blue-400 hover:text-white hover:border-blue-400;
+  }
+  .btn-blue-small.unselected {
+    @apply text-blue-500 bg-white hover:bg-gray-100 hover:text-blue-500;
+  }
+
+  [class*="btn"]:disabled {
+    @apply cursor-not-allowed	bg-gray-300 text-gray-200 border-transparent;
```
Comment (Nebulis) [inline src/tailwind.css:26]: u have an extra space and I only report because you changed the order above :laughing:

File: src/tailwind.css [hunk 2]
```diff
@@ -13,9 +13,16 @@
...
-  .btn-blue-small:hover {
-    @apply text-blue-700 border-blue-500 bg-transparent;
+  .btn-blue-small.selected {
+    @apply text-white bg-blue-500 hover:bg-blue-400 hover:text-white hover:border-blue-400;
+  }
+  .btn-blue-small.unselected {
+    @apply text-blue-500 bg-white hover:bg-gray-100 hover:text-blue-500;
+  }
+
+  [class*="btn"]:disabled {
```
Comment (Nebulis) [inline src/tailwind.css:25]: so smart I always forget this :+1:

File: src/tailwind.css (hunk 1 above)
Comment (simboonlong) [reply src/tailwind.css:27]: where the extra space me no see

File: src/tailwind.css (hunk 2 above)
Comment (simboonlong) [reply src/tailwind.css:26]: turns out need to purge ignore, the purging aggressive leh : (

File: src/tailwind.css (hunk 1 above)
Comment (Nebulis) [reply src/tailwind.css:27]: between allowed and bg-gray. not sure it's an extra space actually but it renders like it on github

Comment (Nebulis) [reply src/tailwind.css:27]: maybe a tab, if you copied from tailwind :)

Comment (simboonlong) [reply src/tailwind.css:27]: strange looks ok on my end, not lying srs 😂

**Review by Nebulis — APPROVED** [review]

**Pull request merged**